azure-automation-plus-dsc-lab/nested/18.26.01.createLnxServer.json

```diff
@@ -0,0 +1,105 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "lnxPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
+      "type": "string"
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "lnxAvSetId": {
+      "type": "string"
+    },
+    "lnx01nicId": {
+      "type": "string"
+    },
+    "saName": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "lnx01name": "[concat(parameters('lnxPrefix'),1)]",
+    "lnxVmSize": "Standard_A1_v2",
+    "imagePublisher": "OpenLogic",
+    "imageOffer": "CentOS",
+    "imageSku": "7.3",
+    "diskNameOs": "[toLower(concat(variables('lnx01name'),'-syst'))]",
+    "saType": "Standard_LRS",
+    "diskNameData": "[toLower(concat(variables('lnx01name'),'-data'))]",
+    "saUrlSuffix": ".blob.core.windows.net",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris

Reply:
Acknowledged. Done.